src/pages/_document.tsx

```diff
@@ -11,7 +11,19 @@ const getInitialProps = async (ctx: DocumentContext): Promise<DocumentInitialPro
 const CustomDocument = (): JSX.Element => {
     return (
         <Html lang="nb-NO">
-            <Head>{/* <link rel="icon" type="image/png" href="/icon.png" /> */}</Head>
+            <Head>
+                <meta name="robots" content="follow, index" />
+                <meta name="msapplication-TileColor" content="#603cba" />
+                <meta name="theme-color" media="(prefers-color-scheme: light)" content="#E6E6E6" />
+                <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1E1E1E" />
```

Comment:
Hva var hensikten med å bytte ut `useColorModeValue` her egt? Det breaker theme-color på ios hvertfall. Vet ikke med android, men å bruke hooken fungerer på ios hvertfall.
```suggestion
const CustomDocument = (): JSX.Element => {
    const themeColor = useColorModeValue('bg.light.primary', 'bg.dark.primary');

    return (
        <Html lang="nb-NO">
            <Head>{/* <link rel="icon" type="image/png" href="/icon.png" /> */}</Head>
            <Head>
                <meta name="robots" content="follow, index" />
                <meta name="msapplication-TileColor" content="#603cba" />
                <meta name="theme-color" content={themeColor} />

```

Reply:
Det funker ikke i det hele tatt på iOS?
Lighthouse meldte at `theme-color` ikke hadde en gyldig verdi, det var nok siden den ble fyllt inn dynamisk med React props. Tenkte at det kunne funke med `prefers-colors-scheme`, var det jeg fant i PWA docs.

Reply:
nvm, det er to forskjellige features.

```js
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#E6E6E6" />
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1E1E1E" />
```
^ dette går etter os-colormode. Men måten vi bruker det på er at det skal gå etter colormode valgt på nettsiden.

Reply:
@bakseter hvis vi skal la det avhenge av os, bør vi kanskje sette `initialColorMode: 'system'`.

Reply:
Men personlig synes jeg det er nice når theme endrer seg med den colormode-knappen :smile:

Reply:
Ait da bare driter vi i det, burde funke nå